tests/test_interface.py

```diff
@@ -0,0 +1,38 @@
+from ttt.interface import Interface
+from ttt.player import Player
+from ttt.computer_player import ComputerPlayer
+from tests.mocks import MockConsole
+from tests.mocks import MockGameRunner
+
+
+def test_starting_a_game_requests_a_welcome_message_be_displayed():
+    runner = MockGameRunner
+    console = MockConsole(input_return='1')
+    interface = Interface(console)
+    interface.start(runner)
+
+    assert console.output_message_call_count == 1
+
+
+def test_starting_a_game_requests_the_game_runner_to_run():
+    runner = MockGameRunner
+    interface = Interface(MockConsole(input_return='1'))
+    interface.start(runner)
+
+    assert interface._runner.run_call_count == 1
+
+
+def test_user_can_select_a_human_opponent():
+    runner = MockGameRunner
+    interface = Interface(MockConsole(input_return='1'))
+    interface.start(runner)
+
+    assert isinstance(interface._runner.player_2, Player)
```

Comment:
Try to avoid using tests for specific types. It's also not particularly good practice to make assertions on private member variables (`_runner`).

Do you think this should be a unit test or an integration test? How regularly might the order of these options change?